src/Scenes/GameScene.js

```diff
@@ -143,15 +180,26 @@ export default class GameScene extends Phaser.Scene {
     this.ropeCurve.draw(this.graphics, 64);
 
     this.recenterCamera();
```

Comment:
I'm getting an error when I reach the end of the level:
Cannot read property 'position' of undefined
    at MatterImage.get [as x] (Transform.js?1165:36)
    at eval (GameScene.js?509e:171)

This is the same error I was getting before when the balloon popped and the face was still looking for it's coords. this is the block of text which solved that before, might still be needed?

Reply:
tyvm